script/Arcueid/Day 9/09_00_ARC09_1.txt

```diff
@@ -157,7 +157,7 @@ No, that's not right.
 [sha:05745a7508e4c6d8f5598d3c09aed72c31ff9f81]{
 -- Page 5, Offset 21323.
 -- この両手は、倒れている<子供|イキモノ>の血で赤いのだ。
-My hands are red. With the blood of a living being, collapsed in a heap on the floor.
+My hands are red. Red with the blood of a living being, collapsed in a heap on the floor.
```

Comment:
Sorry, there were two more changes to make it match nicely - s/a/the and drop the comma
> My hands are red. Red with the blood of **the** living being collapsed in a heap on the floor.